core/src/main/java/com/novoda/noplayer/mediaplayer/CheckBufferHeartbeatCallback.java

```diff
@@ -6,15 +6,15 @@
 
 @SuppressWarnings("PMD.RedundantFieldInitializer")
```

Comment:
this warning probably isn't needed anymore since I don't see any defaults being initialised